### PR TITLE
feat(migration+model): create DB migrations and movie model

### PR DESCRIPTION
**Pull request by SishaarRao**
Why: Migrations help us essentially version control our db schema, and Models represent the data in our db

What: Changes to db config to support migrations and seeding the db for testing purposes, logic+tests for the Movie model object

Details:

Updated eslint versions
Defined scripts to support db migrations using Knex
Defined scripts to seed db
Added seed data (Movies.json)
Created the Movie model
Added unit tests for the Movie model in order to reach 100% test coverage

### CODE REVIEW

**Comment by SishaarRao** [comment]
Apologies for all the commits - for some reason I was unable to trigger Travis on my commits to fix postgresql not starting, so I was trying a few different things.
Turns out the issue was in my travis.yml file:
Reordering the file from
```json
...
before_script:
  - yarn db:setup:user
  - yarn db:reset
services:
  - postgresql
script:
  - yarn test
  - yarn enforce
  - yarn lint
```
to
```json
...
services:
  - postgresql
before_script:
  - yarn db:setup:user
  - yarn db:reset
script:
  - yarn test
  - yarn enforce
  - yarn lint
```
seemed to fix the issue.
Why?    ¯\_(ツ)_/¯

**Comment by Lianathanoj** [comment]
yeah, i think because travis CI requires the postgres config to already be setup, the `postgresql` service needs to be before the yarn scripts in order to actually trigger the scripts.

one other thing: could you update your PR title to reflect the practices we use [here](https://www.notion.so/lob/Code-Review-Guidelines-aad42d64cb724f3cadd1ab233bfc93df#478e0798ee32403296a709b2897420a7)?

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,12 @@ node_js:
...
+services:
+  - postgresql
+before_script:
+  - yarn db:setup:user
+  - yarn db:reset
 script:
   - yarn test
-  - yarn lint
+  - yarn enforce
+  - yarn lint
```
Comment (Lianathanoj) [inline .travis.yml]: nit: could you move the `yarn lint` to before the `yarn test`? this doesn't really matter for this specific project, but it's a good practice b/c some repos which contain thousands of test suites may take, say, 15+ minutes to run. Catching a potentially minor linting error at the end would necessitate fixing that error, deploying that change, and re-running the entire testing suite for another 15+ minutes. The time it takes to run the linter would comparatively be much faster than it takes to run the test, so If we caught the linting error at the beginning that would def be more worthwhile as we prefer to fail fast

Comment (SishaarRao) [reply .travis.yml]: That makes sense, I pushed a new change to resolve this!

**Review by Lianathanoj — COMMENTED** [review]
Sorry, I should've pointed you to a more exact URL: see [here](https://www.notion.so/lob/Code-Review-Guidelines-aad42d64cb724f3cadd1ab233bfc93df#348eb8129e2a448e9de5bacd9a6f837e)

So in this case, we want to use something like `feat(...): ...`. This might be slightly weird b/c your changes encompass multiple changes which I would usually split into multiple PRs (e.g. 1 PR for setting up the project base and another to setup the model), so it would look something like `feat(movies): create migration and movie model` when usually we want something like `feat(movies): create migration` and `feat(movies): create model`, but right now it's fine. Just make sure to split up the steps going forward!

**Review by Lianathanoj — APPROVED** [review]
LGTM pending the PR title change!

**Pull request merged**